src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -191,19 +196,30 @@ public void applyMergeReview(MergeReview mergeReview) throws ServiceException {
 		}
 
 		// Perform rebase merge
-		ObjectReader conceptReader = objectMapper.readerFor(Concept.class);
 		ManuallyMergedConcept manuallyMergedConcept = null;
 		List<Concept> concepts = new ArrayList<>();
+		ObjectReader conceptReader = objectMapper.readerFor(Concept.class);
 		try {
 			for (ManuallyMergedConcept mmc : manuallyMergedConcepts) {
 				manuallyMergedConcept = mmc;
-				Concept concept;
-				if (manuallyMergedConcept.isDeleted()) {
+				Concept concept = null;
+
+				String conceptJson = manuallyMergedConcept.getConceptJson();
+				if (conceptJson != null) {
+					concept = conceptReader.readValue(conceptJson);
+				}
+
+				Concept sourceConcept = conceptService.find(String.valueOf(manuallyMergedConcept.getConceptId()), mergeReview.getSourcePath());
+				boolean sourceConceptVersioned = sourceConcept != null && sourceConcept.isReleased() && sourceConcept.getEffectiveTime() != null;
+				if (sourceConceptVersioned && concept == null) {
+					concept = sourceConcept;
+				} else if (sourceConceptVersioned && !concept.isReleased() && concept.getEffectiveTime() == null) {
+					concept = autoMergeConcept(sourceConcept, concept);
+				} else if (manuallyMergedConcept.isDeleted()) {
```

Comment:
During the rebase, I am finding the parent version of the Concept. If the parent version has been versioned and the child isn't, then I am calling the `autoMergeConcept` function which restores our fields.